porch/func/server/server.go

```diff
@@ -15,20 +15,42 @@
 package main
 
 import (
+	"flag"
 	"fmt"
 	"net"
 
 	pb "github.com/GoogleContainerTools/kpt/porch/func/evaluator"
+	"github.com/GoogleContainerTools/kpt/porch/func/internal"
 	"google.golang.org/grpc"
+	"k8s.io/klog/v2"
+)
+
+var (
+	port      = flag.Int("port", 9445, "The server port")
+	functions = flag.String("functions", "./functions", "Path to cached functions.")
+	config    = flag.String("config", "./config.yaml", "Path to the config file.")
 )
 
 func main() {
-	lis, _ := net.Listen("tcp", fmt.Sprint(":9445"))
-	server := grpc.NewServer()
-	pb.RegisterFunctionEvaluatorServer(server, &evaluator{})
-	server.Serve(lis)
-}
+	flag.Parse()
 
-type evaluator struct {
-	pb.UnimplementedFunctionEvaluatorServer
+	address := fmt.Sprintf(":%d", *port)
+	lis, err := net.Listen("tcp", address)
+	if err != nil {
+		klog.Fatalf("failed to listen: %v", err)
```

Comment:
Personal style nit: I just write some boilerplate

```
func main() {
  err := run(context.Background())
  if err != nil {
    fmt.Fprintf(os.Stderr, "unexpected error: %v\n", err)
    os.Exit(1)
  }
}

func run(ctx context.Context) error {
...
```

It makes it easier to move code around, because we don't have a special function (main) that doesn't follow the normal error handling rules.

Reply:
done